db/entity/DiscordMember.ts

```diff
@@ -25,10 +25,16 @@ export class DiscordMember {
   discordMemberId: string;
 
   @Column()
-  discordServerId: string;
+  DiscordServerId: string;
 
-  @ManyToOne((type) => DiscordServer, (server) => server.members)
-  discordServer: DiscordServer;
+  @Column()
+  DiscordServerConfigId: string;
+
+  @ManyToOne(
+    (type) => DiscordServerConfig,
+    (serverConfig) => serverConfig.members
+  )
+  DiscordServerConfig: DiscordServerConfig;
```

Comment:
I'm not sure this is a good idea to attach the `discordMember`  to a `discordServerConfig` => let's discuss but keeping the `discordServer` model and adding a new `discordServerConfig` model seems cleaner to me, we should not duplicate the `discordMember` for each config!